engine/tests/story/test_player_concept.py

```
@@ -0,0 +1,117 @@
+
+
+def _load_regent_cls():
+    """Import the world-local Regent the way the bundle exposes it."""
+    pkg_root = (
+        Path(__file__).resolve().parents[3]
+        / "worlds"
+        / "coronate_the_regent"
+    )
+    if str(pkg_root) not in sys.path:
+        sys.path.insert(0, str(pkg_root))
+    from coronate_the_regent.domain import Regent  # noqa: WPS433
```

Ruff (0.15.12)

[warning] Invalid rule code in `# noqa`: WPS433

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)